security/grpc.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
```

Comment:
I actually think there should be two packages. 

- `jwtutil` - in addition to validating JWT, some libs from JVS could be relocated over (e.g. build/sign JWT)
- `grpcutil` -  a package for useful "middlewares", and the this would be `grpcutil.JWTAuthenticator` with a func `Authenticate(ctx) (string, error)`

Reply:
I don't feel strongly about this, so i'm happy to break them into separate packages if you think its best.